converter/isosts2nisosts/isosts2nisosts.xsl

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:mml="http://www.w3.org/1998/Math/MathML"
+  xmlns:tbx="urn:iso:std:iso:30042:ed-1"
+  xmlns:xhtml="http://www.w3.org/1999/xhtml"
+  xmlns:mle="http://www.iso.org/ns/mle"
+  xmlns:isosts="http://www.iso.org/ns/isosts"
+  xmlns:loc="http://www.iso.org/ns/localization"
+  exclude-result-prefixes="xs xlink mml tbx xhtml isosts loc mle"
+  version="2.0">
+
```

Comment:
please remove namespaces mle, loc, isosts. They are not used anywhere in our pure ISOSTS XML